sites/cnfpt/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Changed
+
+- Unpin Django to 3.1.1 since release of djangocms-admin-style 2.0.2
```

Comment:
Unpin to sounds weird

Reply:
Maybe unpin Django now the django-admin-style 2.0.2 supports the latest version 3.1.3